crates/apub/src/objects/post.rs

```diff
@@ -234,13 +235,15 @@ impl Object for ApubPost {
 
       let body = read_from_string_or_source_opt(&page.content, &page.media_type, &page.source);
       let body = process_markdown_opt(&body, slur_regex, context).await?;
+      let alt_text = process_markdown_opt(&page.alt_text, slur_regex, context).await?;
```

Comment:
Not sure what the `read_from_string_or_source` is here, or how it would apply to another optional markdown field alongside body.

Reply:
Thats irrelevant here as alt text should be plaintext.

Reply:
Ah, I'd assumed it was a full markdown type field. I'll change this.